rasa/core/training/interactive.py

```diff
@@ -655,22 +655,37 @@ def _slot_history(tracker_dump: Dict[Text, Any]) -> List[Text]:
 def _request_export_info() -> Tuple[Text, Text, Text]:
     """Request file path and export stories & nlu data to that path"""
 
+    def is_not_empty(path: Text) -> bool:
+        return path is not None and path != ""
+
+    def validate_story_export_file(path: Text) -> bool:
+        return is_not_empty(path) and path.endswith(".md")
+
+    def validate_nlu_export_file(path: Text) -> bool:
+        return is_not_empty(path) and (path.endswith(".md") or path.endswith(".json"))
+
+    def validate_domain_export_file(path: Text) -> bool:
+        return is_not_empty(path) and (path.endswith(".yaml") or path.endswith(".yml"))
+
     # export training data and quit
     questions = questionary.form(
         export_stories=questionary.text(
             message="Export stories to (if file exists, this "
             "will append the stories)",
             default=PATHS["stories"],
+            validate=validate_story_export_file,
```

Comment:
Maybe we could even replace these with e.g.:
```
validate=Validator.from_callable(validate_story_export_file, error_message='Stories file must end in .md'), 
```
to make use of more descriptive error msgs than `invalid input`

Source here: https://github.com/prompt-toolkit/python-prompt-toolkit/blob/c7851c09ffbfa7d06458408abee77ab6f548852d/prompt_toolkit/validation.py#L84

Reply:
Cool suggestion, will do!

Reply:
had to do it a bit differently since questionary doesn't support objects of this type. Will do a pr there :-)